src/rpc/rpccategory.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2010 Satoshi Nakamoto
+// Copyright (c) 2009-2018 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_RPC_RPCCATEGORY_H
+#define BITCOIN_RPC_RPCCATEGORY_H
+
+#include <string>
+
+enum class RPCCategory
+{
+    blockchain = 0,
```

Comment:
I don't think you have to set this to `0`, also compile time constants should be `ALL_UPPER_CASE`.